requirements.txt

```diff
@@ -0,0 +1,13 @@
+#authlib~=1.2.1
+boto3~=1.26.159
+botocore~=1.29.159
+#cryptography~=41.0.3
+flask~=2.3.3
+flask-cors
+govuk-frontend-jinja
+gunicorn
```

Comment:
I know we don't do this at the moment, but we should be fixing our dependencies to specific versions - maybe something else to add to the Web Application Standards session 🤔

Reply:
will do in next PR